src/components/timeline/index.css

```diff
@@ -118,4 +121,15 @@
   border-bottom: 1px solid var(--grey-30);
   margin-right: calc(var(--vertical-scrollbar-reserved-width) * -1);
   margin-left: calc(var(--thread-label-column-width) * -1);
+  background: white;
```

Comment:
Can you please just add a comment for this background, explaining it needs to be opaque so that it properly hides the hidden tracks container border below?